<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Car UI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_main">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="homePage">
       <layout class="QHBoxLayout" name="horizontalLayout_home">
        <item>
         <widget class="QPushButton" name="btn_media">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Media</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_navigation">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Navigation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="mediaPage">
       <layout class="QVBoxLayout" name="verticalLayout_media">
        <item>
         <widget class="QLabel" name="label_playlist_header">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Playlist</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="list_playlist">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_playback_info">
          <item>
           <widget class="QLabel" name="label_now_playing_title">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Now Playing:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_current_track_info">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>---</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_playback_info">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_elapsed_duration">
          <item>
           <widget class="QLabel" name="label_elapsed_time">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_progress">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_duration">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>00:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_controls">
          <item>
           <spacer name="horizontalSpacer_controls_left">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btn_previous">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&lt;&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_play_pause">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_next">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>&gt;&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_controls_right">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_volume">
          <item>
           <widget class="QLabel" name="label_volume">
            <property name="text">
             <string>Volume:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_volume">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_media">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="navigationPage">
       <layout class="QVBoxLayout" name="verticalLayout_navigation">
        <item>
         <widget class="QLabel" name="label_nav_placeholder">
          <property name="text">
           <string>Navigation Screen (Map Placeholder)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <customwidgets/>
 <resources/>
</ui>
